tests/Integrations/Greenhouse/GreenhouseTest.php

```diff
@@ -0,0 +1,501 @@
+<?php
+
+/**
+ * Tests for Greenhouse integration file.
+ */
+
+namespace EightshiftFormsTests\Integrations\Greenhouse;
+
+use Brain\Monkey\Functions;
+use EightshiftForms\Hooks\Filters;
+use EightshiftFormsTests\BaseTest;
+use EightshiftFormsTests\Mocks\MockGreenhouse;
+class GreenhouseTest extends BaseTest
+{
+	protected function _inject(DataProvider $dataProvider)
+	{
+		$this->dataProvider = $dataProvider;
+	}
+
+	protected function _before()
+	{
+		parent::_before();
+		$this->greenhouse = $this->diContainer->get(MockGreenhouse::class);
+	}
+
+	/**
+	 * Post application - Success
+	 *
+	 * @return void
+	 */
+	public function testSuccessPostingGreenhouseApplication()
+	{
+		$this->addHooks();
+
+		Functions\stubs(
+			[
+				'wp_remote_post' => function() {
+					return DataProvider::successGreenhouseApplicationResponse();
+				},
+			]
+		);
+
+		$params = DataProvider::greenhouseApplicationParams();
+
+		$response = $this->greenhouse->postGreenhouseApplication(
+			DataProvider::MOCKED_GH_APPLICATION_SUCCESS_URL,
+			$params
+		);
+
+		$this->assertEquals($response['success'], 'Candidate saved successfully');
```

Comment:
```suggestion
		$this->assertSame($response['success'], 'Candidate saved successfully');
```

Always use `assertSame`, because it will check the type and the value. You can fix this in the entire test suite.

Reply:
👍